catalog/cloudtrail-policies.yaml

```diff
@@ -1,3 +1,11 @@
+- sid: "DenyCloudTrailActions"
+  effect: "Deny"
+  actions:
+    - "cloudtrail:DeleteTrail"
+    - "cloudtrail:PutEventSelectors"
+    - "cloudtrail:StopLogging"
+    - "cloudtrail:UpdateTrail"
+    
```

Comment:
need 

```
resources:
    - "*"
```

also, can you maybe combine these two policies?
or clean them up since they gave overlapping actions